### PR TITLE
Updated User object

### PR DESCRIPTION
**Pull request by zplata**
Note: until the parser can parse the new User object, don't try testing it. Will result in compilation errors.

### CODE REVIEW

**Comment by WillHyde** [comment]
I got you. Want me to parse it for you on a new branch?

File: RoomMe/app/src/main/java/com/example/trocket/roomme/User.java [hunk 1]
```diff
@@ -3,32 +3,54 @@
...
+        this.userID = userID;
         fullName = name;
+        this.gender = gender;
         this.age = age;
+        this.phoneNumber = phoneNumber;
+        this.email = email;
+        this.status = status;
+        this.housingPrice = housingPrice;
+        this.bio = bio;
+        this.favoritesUserIDS = favoritesUserIDS;
```
Comment (jimmyhlee94) [inline RoomMe/app/src/main/java/com/example/trocket/roomme/User.java]: typo. this property should be `favoritedUserIDS`

File: RoomMe/app/src/main/java/com/example/trocket/roomme/User.java [hunk 2]
```diff
@@ -3,32 +3,54 @@
...
+    public int getUserID() { return userID; }
     public String getName() { return fullName; }
+    public int getGender() { return gender; }
     public int getAge() { return age; }
     public String getPhoneNumber() { return phoneNumber; }
     public String getEmail() { return email; }
+    public int getStatus() { return status; }
+    public double getHousingPrice() { return housingPrice; }
+    public String getBio() { return bio; }
+    public List<Integer> getFavoritesUserIDS() { return favoritesUserIDS; }
```
Comment (jimmyhlee94) [inline RoomMe/app/src/main/java/com/example/trocket/roomme/User.java]: here too.

File: RoomMe/app/src/main/java/com/example/trocket/roomme/User.java [hunk 3]
```diff
@@ -42,15 +64,30 @@ public int describeContents() {
...
+        this.userID = in.readInt();
+        this.fullName = in.readString();
+        this.gender = in.readInt();
+        this.age = in.readInt();
+        this.phoneNumber = in.readString();
+        this.email = in.readString();
+        this.status = in.readInt();
+        this.housingPrice = in.readDouble();
+        this.bio = in.readString();
+        this.favoritesUserIDS = new ArrayList<Integer>();
```
Comment (jimmyhlee94) [inline RoomMe/app/src/main/java/com/example/trocket/roomme/User.java]: _sigh_

**Comment by jimmyhlee94** [comment]
https://www.youtube.com/watch?v=07So_lJQyqw

**Pull request merged**